src/django/api/serializers/facility/mit_living_wage_download_helper.py

```
@@ -0,0 +1,82 @@
+        self.__provider = MITLivingWageProvider()
+        self.__partner_field_resolved = False
+        self.__partner_field: Optional[PartnerField] = None
+
+    def get_cells(self, facility) -> List[str]:
+        '''
+        Return `[county_link, county_link_text]` for the given facility,
+        or two empty strings if the provider has no data for it.
+        '''
+        try:
+            data = self.__provider.fetch_data(facility)
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)